src/mol-theme/color/volume-value.ts

```diff
@@ -87,12 +87,12 @@ export function VolumeValueColorTheme(ctx: ThemeDataContext, props: PD.Values<Vo
                 }
 
                 const value = getTrilinearlyInterpolated(location.position);
-                if (isNaN(value)) return props.defaultColor;
+                if (isNaN(value)) return 0xFFFFFF as Color; // props.defaultColor;
```

Comment:
Add an `UndefinedColor` constant since this refers to `reserve (1 << 24) - 1 for undefiend values` above?

Reply:
actually just forgot to use props defaultColor, the comment was from before the prop existed